state/cassandra/metadata.yaml

```
@@ -0,0 +1,75 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: state
+name: cassandra
+version: v1
+status: stable
+title: "Apache Cassandra"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-cassandra/
+capabilities:
+  # If actorStateStore is present, the metadata key actorStateStore can be used
+  - crud
+  - ttl
+authenticationProfiles:
+  - title: "Username and password"
+    description: "Authenticate using username and password."
+    metadata:
+      - name: username
+        type: string
+        required: true
+        description: "The username of database user."
+        example: '"admin"'
+      - name: password
+        type: string
+        required: true
+        sensitive: true
+        description: "The password of the user."
+        example: '"password"'
+      - name: hosts
+        type: string
+        required: true
+        description: "Comma separated list of the hosts."
+        example: '"cassandra.cassandra.svc.cluster.local,other.cassandra.example.tld"'
+metadata:
+  - name: port
+    type: number
+    description: "Port for communication."
+    default: "9042"
+    example: "8080"
```

You know what, let's move hosts out from authentication.

```suggestion
metadata:
  - name: hosts
    type: string
    required: true
    description: "Comma separated list of the hosts."
    example: '"cassandra.cassandra.svc.cluster.local,other.cassandra.example.tld"'
  - name: port
    type: number
    description: "Port for communication."
    default: "9042"
    example: "8080"
```